test/remote/password_forgot_tests.js

```diff
@@ -31,6 +31,7 @@ TestServer.start(config)
         .then(
           function (x) {
             client = x
+            client.setAllowedOrigin(config.corsOrigin)
```

Comment:
Is it necessary to set this on all the client objects here, or just for completeness?  If necessary, it seems like something we should refactor so that it happens in a single place.